buildpacks/ruby-bundler/README.md

```diff
@@ -0,0 +1,12 @@
+# Sample: Ruby Bundler Buildpack
+
+Compatible apps:
+- Ruby apps that use Bundler
+
+### Usage
+
+```bash
+pack build sample-app --builder <builder> --buildpack . --path ../../apps/ruby-bundler/
```

Comment:
as with the other PR, I think this is a little risky to recommend. might want to link them to the docs.

Reply:
I'm going to merge the PR as is and reach out to you via slack to discuss this more in detail. I want to keep it as simple as possible but I understand your concern.